examples/Org.OData.Capabilities.V1.capabilities.md

```diff
@@ -0,0 +1,85 @@
+# Annotating capabilities
+
+Consider a `Headers` entity set with contained entity sets `Headers(...)/Items` and `Headers(...)/Items(...)/Subitems`. Each entity type has a server-generated `uuid` and a `canUpdate` property. The header has an additional `canInsertItems` and the item has `canInsertSubitems` which govern insertability. Insertability of headers depends on the permissions contained in a `Permissions` singleton. The header also has a `canUpdateSubsetOfItems` which governs updatability with `/$filter(...)/$each` (the analogous thing on item level is omitted from this example).
+
+The following annotations express insertability and updatability in terms of these properties. Deletability is analogous to updatability but omitted from this example.
+
+For the header level, the entity set is annotated directly.
+
+```jsonc
+"self.Container/Headers": {
+  "@Capabilities.InsertRestrictions": {
+    "Insertable": {"$Path": "/self.Container/Permissions/canInsertHeaders"},
+    "NonInsertableProperties": ["uuid"]
+  },
+  "@Capabilities.UpdateRestrictions": {
+    "Updatable": {"$Path": "canUpdate"},
+    "NonUpdatableProperties": ["uuid"]
+  }
+}
+```
+
+The item level has no named entity set. It is annotated using `NavigationRestrictions` on header level and `InsertRestrictions` and `UpdateRestrictions` on item level. When a property could be expressed either on the deeper item level or on the higher-up header level, the deeper level is generally preferred: For example, the `NonUpdatableProperties` property in the deeper `UpdateRestrictions` is favored over its commented-out namesake in the higher-up `NavigationRestrictions`, and the commented-out `Updatable` is even invalid, because the instance path to `canUpdate` is collection-valued. But the higher-up `InsertRestrictions/Insertable` and `UpdateRestrictions/FilterSegmentSupported` cannot be avoided, because the instance paths to `canInsertItems` and `canUpdateSubsetOfItems` must be evaluated on header level.
+
+```jsonc
+"self.Container/Headers": {
+  "@Capabilities.NavigationRestrictions": {
+    "RestrictedProperties": [{
+      "NavigationProperty": "Items",
+      "InsertRestrictions": {
+     // "NonInsertableProperties": ["Items/uuid"],
+        "Insertable": {"$Path": "canInsertItems"}
+      },
+      "UpdateRestrictions": {
+     // "Updatable": {"$Path": "Items/canUpdate"},
+     // "NonUpdatableProperties": ["Items/uuid"],
+        "FilterSegmentSupported": {"$Path": "canUpdateSubsetOfItems"}
+      }
```

Comment:
This shows that the decision where to annotate something is made per property (such as `UpdateRestrictions/Updatable`), not per term (such as `UpdateRestrictions`).